### PR TITLE
Secondary command names

### PR DESCRIPTION
**Pull request by zacharytang**
Added secondary keywords for most commands. Updated docs to reflect aliases and secondary keywords, and added tests to test alternative keywords.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -70,12 +72,14 @@ Examples:
 === Listing all persons : `list`
 
 Shows a list of all persons in the address book. +
-Format: `list`
+Format: `list` +
+Alternatives: `l` , `showall`
```
Comment (April0616) [inline docs/UserGuide.adoc]: ‘viewall', 'view'

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -70,12 +72,14 @@ Examples:
...
 
 === Editing a person : `edit`
 
 Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
+Alternatives: `e` , `modify`
```
Comment (April0616) [inline docs/UserGuide.adoc]: 'change', ???

**Review by April0616 — APPROVED** [review]
generally good!!

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 3]
```diff
@@ -117,6 +117,146 @@ public void parseCommand_select() throws Exception {
     }
 
     @Test
+    public void parseCommandAlias_add() throws Exception {
```
Comment (April0616) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:120]: method name should not contain an 'underscore'

**Review by April0616 — DISMISSED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (zacharytang) [reply docs/UserGuide.adoc]: "viewall" can be implemented, but I feel that "view" overlaps with the functionality of the select command

Comment (April0616) [reply docs/UserGuide.adoc]: oh yes

**Review by nbriannl — APPROVED** [review]
Besides underscored test method name. Yuting reviewed everything is good.

**Pull request merged**